### PR TITLE
WIP: Relax ConsulKV tests with retries on dependent operations

### PR DESCRIPTION
**Pull request by perryibm**
PG build 152.  Travis itself it the test so leaving as WIP until I get a few green from Travis.

### CODE REVIEW

File: tests/src/whisk/consul/ConsulKVTests.scala [hunk 1]
```diff
@@ -35,19 +36,32 @@ class ConsulKVTests extends FlatSpec with Matchers {
     private val config = new WhiskConfig(requiredProperties)
     private val consul = new ConsulKV(config.consulServer)
 
+    /*
```
Comment (ioana-blue) [inline tests/src/whisk/consul/ConsulKVTests.scala]: The retry method looks like something general that we could have in a utility and reuse elsewhere in tests. No?

Comment (rabbah) [inline tests/src/whisk/consul/ConsulKVTests.scala]: see `whisk.utils.retry`

**Comment by perryibm** [comment]
It's passed twice already - on its third now.  Travis build # does not increment when rebuilding.

**Comment by rabbah** [comment]
edited slightly and merged.

**Pull request closed without merging**